resources/integration.rb

```diff
@@ -30,11 +36,16 @@
     return
   end
 
-  Chef::Log.debug("Removing integration #{new_resource.name}")
+  Chef::Log.debug("Removing integration #{new_resource.property_name}")
 
   execute 'integration remove' do
-    command   "\"#{agent_exe_filepath}\" integration remove #{new_resource.name}"
+    command   "\"#{agent_exe_filepath}\" integration remove #{new_resource.property_name}"
     user      'dd-agent' unless node['platform_family'] == 'windows'
+
+    not_if {
+      output = shell_out("#{agent_exe_filepath} integration show -q #{new_resource.property_name}").stdout
+      output.empty?
```

Comment:
don't we need to `strip` the output before checking if it's empty?

Reply:
I've added the `strip` call and revalidated in all Chef versions mentioned in the PR! 👍 
Merging.